sklearn/model_selection/_split.py

```diff
@@ -891,6 +901,8 @@ def get_n_splits(self, X, y, groups):
         """
         if groups is None:
             raise ValueError("The groups parameter should not be None")
+        groups = check_array(groups, ensure_2d=False, dtype=None)
+        X, y, groups = indexable(X, y, groups)
```

Comment:
Aside: should we implement a default `get_n_splits` that returns the length of `self.split(X, y, groups))`?